lib/active_model/serializer/field.rb

```diff
@@ -0,0 +1,56 @@
+module ActiveModel
+  class Serializer
+    # Holds all the meta-data about a field (i.e. attribute or association) as it was
```

Comment:
still think it should be e.g., but since it's intentional on your part and it's super unimportant, ok to merge when green